dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -146,6 +154,11 @@ impl Display for MaybeSledId {
     }
 }
 
+/// Format a chrono::DateTime like "2018-01-26T18:30:09.453Z"
+fn chrono_to_rfc_3999(time: &chrono::DateTime<chrono::Utc>) -> String {
+    time.to_rfc3339_opts(SecondsFormat::Millis, true)
```

Comment:
I think this duplicates `datetime_rfc3339_concise()`.

Also, the name is slightly wrong (3999 vs 3339).